src/server/server.js

```diff
@@ -78,6 +78,15 @@ if (env === 'development') {
     );
 }
 
+if (process.env.NODE_ENV === 'production') {
+    const resolvedAssets = require(path.join(
+        __dirname,
+        '../..',
+        '/tmp/webpack-stats-prod.json'
+    ));
+    const supportedLocales = getSupportedLocales();
```

Comment:
same thing here -- if supportedLocales is defined as false and then conditionally set to the output of getSupportedLocales then we can scrap the conditional mess below